IOS/SwiftSeedProject/SwiftSeedProject/Domain/Article.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Article.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 13/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class ArticleM {
```

Comment:
We can remove this class as we are now using Core Data entities. @mvazquez-ms is it ok to delete it?